v5/core/core.go

```diff
@@ -105,10 +107,30 @@ func (core *Core) Setup(sdkSettings model.SdkSettings, deviceProperties model.De
 		core.environment = client.NewSaasEnvironment(envApi)
 	}
 
-	// TODO Analytics.Analytics.Initialize(deviceProperties.RolloutKey, deviceProperties)
+	analyticsDisabled := (roxOptions != nil && roxOptions.IsAnalyticsReportingDisabled()) ||
+		roxyPath == "" ||
```

Comment:
Mm..
Isn't analytics should be disabled when Roxy is on
So roxyPath != ""
?

Reply:
Yea, you're correct. I moved that out of a `!()` and forgot to distribute the `!`